test/foundry/README.md

```diff
@@ -0,0 +1,27 @@
+# JetStaking Foundry tests
+
+This repository contains foundry tests for `jetStakingV1.sol` and `Treasury.sol` contract.
+
+## Install Foundry
+
+Prerequisites:
+
+Run the following command in the terminal:
+
+```bash
+curl -L https://foundry.paradigm.xyz | bash
+```
+
+This will download foundryup. Then install Foundry by running:
+
+```bash
+foundryup
+```
+
+## Compile and test
+
+Compile the contracts and run the tests using the following command:
+
+```bash
+yarn run forge-test
```

Comment:
Tried to follow this README file and it fails to compile prior running the tests:

```
Compiler run failed
error[6275]: ParserError: Source "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol" not found: File not found.
 --> test/foundry/Regular_Tests/Setup.sol:6:1:
  |
6 | import "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^



error[6275]: ParserError: Source "openzeppelin-contracts/contracts/token/ERC20/ERC20.sol" not found: File not found.
 --> test/foundry/Regular_Tests/Setup.sol:7:1:
  |
7 | import "openzeppelin-contracts/contracts/token/ERC20/ERC20.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
To resolve this issue:
User should follow these steps:

1- install OpenZeppelin contracts
```bash
forge install openzeppelin/openzeppelin-contracts
forge install openzeppelin/openzeppelin-contracts-upgradeable
```
2- Configure remappings as mentioned [here](https://github.com/aurora-is-near/aurora-staking-contracts/pull/170/files#r970732523).